command.go

```diff
@@ -185,6 +188,38 @@ func parseAddArgs(args []string) (Command, error) {
 	return e, nil
 }
 
+func parseConsoleArgs(args []string) (Command, error) {
+	flag := pflag.NewFlagSet("console", pflag.ContinueOnError)
+	flag.String("assume", "", "Role to assume")
+	flag.Duration("duration", 0, "Duration (15m-12h)")
+	flag.Usage = func() {}
+	err := flag.Parse(args)
+	if err != nil {
+		return nil, err
+	}
+
+	vaultName := ""
+	assume, _ := flag.GetString("assume")
+	duration, _ := flag.GetDuration("duration")
+	if duration != 0 && (duration < ConsoleMinDuration || duration > ConsoleMaxDuration) {
```

Comment:
`Console*Duration` constants aren't specific to the console, but to the underlying Role.  These should probably be named/refactored accordingly.